src/main/java/laddarGame/domain/Ladder.java

```diff
@@ -32,4 +33,13 @@ public Lines(List<Line> lines) {
                 .map(Line::lineDto)
                 .collect(toList());
     }
+
+    public List<String> play(Players players) {
+        final Players[] result = {players};
```

Comment:
```suggestion
        final Players finalPlayers = players;
```
이런식으로 final 변수로 다시 정의만 해도 된답니다1